frontend/common/stores/account-store.js

```diff
@@ -367,20 +367,15 @@ const controller = {
         { hosted_page_id: hostedPageId },
       )
       .then((res) => {
-        const idx = _.findIndex(store.model.organisations, {
-          id: store.organisation.id,
+        try {
+          if (res && res.subscription && res.subscription.plan) {
+            API.trackEvent(Constants.events.UPGRADE(res.subscription.plan))
+            API.postEvent(res.subscription.plan, 'chargebee')
+          }
+        } catch (e) {}
```

Comment:
I don't really love the empty catch here. Can we silently swallow error conditions that we've allowlisted ahead of time?

Reply:
This is just in case Analytics tools are broke for whatever reason, in which case we'd still see something in sentry. Just needs to ensure the later more important things happen regardless.